### DISCUSSION TITLE
Add headers in between regular entries

### DISCUSSION THREAD

**Discussion by fdw** [davatorium/rofi#1694] [Ideas/Feature Requests]
I'd like to structure entries a bit more, so that instead of one big list, there would be headers in between, separating different groups. As far as I've seen, this is not possible yet. Am I wrong?

I've thought about just adding them as regular entries, but then they could be selected, which is less than ideal.

Do you think such a feature would be useful enough to be added?

**Comment by DaveDavenport** [comment]
Yes, and it already exists.

```
echo -en "aap\0nonselectable\x1ftrue\n" | rofi -dmenu
```
creates an unselectable entry 'aap'.

**Reply by fdw** [reply]
That's very cool! Thanks! 🙂
